deeposlandia/webapp.py

```diff
@@ -136,15 +136,16 @@ def demo_prediction():
     filename = request.args.get('img')
     filename = os.path.join("deeposlandia", filename[1:])
     dataset = request.args.get('dataset')
+    agg_value = dataset == "mapillary"
```

Comment:
Does it mean that, for the webapp, the mapillary dataset is necessarily with aggregated labels?

Reply:
For now, yes (the result are better...). However we could imagine in a near future the management of both cases, with a checkbox on the Mapillary webpages!

Reply:
Very good idea.